src/Standards/Generic/Tests/CodeAnalysis/EmptyStatementUnitTest.inc

```diff
@@ -72,3 +72,19 @@ try {
 if (true) {} elseif (false) {}
 
 match($foo) {};
+
+// phpcs:set Generic.CodeAnalysis.EmptyStatement allowComment[] elseif,catch
```

Comment:
When using custom properties in test case files, it is good practice to reset the property to the default value after the test cases to which it applies. This helps prevent future test cases being added running into an unexpected situation.